newton/examples/example_mujoco.py

```diff
@@ -251,7 +251,7 @@ def __init__(
             builder = Example.create_model_builder(robot, num_envs, randomize, self.seed)
 
         # finalize model
-        self.model = builder.finalize()
+        self.model = builder.finalize(requires_grad=False)
```

Comment:
`requires_grad` is False by default here, I think it would be better to not pass it explicitly in these examples then. The user might think they always have to do this when in reality gradient computation is something that needs to be _enabled_.